src/main/fc/rc.h

```diff
@@ -47,3 +47,7 @@ float applyCurve(int axis, float deflection);
 bool getShouldUpdateFeedforward();
 void updateRcRefreshRate(timeUs_t currentTimeUs);
 uint16_t getCurrentRxRefreshRate(void);
+
+#ifdef USE_FEEDFORWARD
```

Comment:
Declarations in header files are generally not made conditional in Betaflight - there is no negative impact for not doing so.

Reply:
Conditional removed